src/Member.java

```diff
@@ -0,0 +1,189 @@
+        } else {
+            userList.add(members);
+            System.out.println("New Member ["+newUser_Name.toUpperCase()+"] has been added to the Library");
+        }
+
+    }
+
+    public void addNewAdmin(String newAdmin_Name, String newAdmin_Password) {
+        Member members = new Member(newAdmin_Name, newAdmin_Password, Role.ADMIN);
+
+        //userList = new ArrayList<Member>();
+        if (userList.contains(newAdmin_Name)) {
+            System.out.println("User already Exists !! Please try another user name.");
+        } else {
+            userList.add(members);
+            
+        }
+    }
+
+    public void login(String logUserName, String logPassword) {
+        for (int i = 0; i <= userList.size(); i++) {
+            if (logUserName.equals(userList.get(i).getUserName()) && logPassword.equals(userList.get(i).getPassword())) {
+                String logRole = userList.get(i).getRole();
+                System.out.println(logRole);
+                if (logRole.equals("ADMIN")) {
+                    //todo some code
+                    System.out.println("Logged in as ADMIN Successfully !!");
+                    String inputFromAdmin;
+                    do {
+
+                        Scanner sc = new Scanner(System.in);
+                        System.out.println("Select Number from bellow to perform Action:");
+                        System.out.println("1. Add New Member");
+                        System.out.println("2. Add New Book");
+                        System.out.println("3. Search Book");
+                        System.out.println("4. Logout");
+                        inputFromAdmin = sc.nextLine();
+                        
+                        switch(inputFromAdmin){
+                            case "1":
+                                Scanner scanner = new Scanner(System.in);
+                                System.out.println("Enter New Member Name: ");
+                                String newMemberName = scanner.nextLine();
+                                System.out.println("Enter Password for new Member: ");
+                                String newMemberPassword = scanner.nextLine();
+                                
+                                addNewUser(newMemberName, newMemberPassword);
+                                
+                                
+                            case "2":
+                                // add some new books
+                            case "3":
+                                // search functionalities
+                            case "4":
+                                LibraryApp.getUserInforLogin();
+                        }
+                        
+
+                    } while (true);
+                    //do something 
+                    // assigning operation
+
+                    
+
+                } else if (logRole.equals("USER")) {

```

Comment:
I have a question here; if I write like this

User user = null;  //what would this null object do?

also the same thing you explained here; could you provide some link for further reading about this very specific thing @dfoulkes